src/datomic_toolbox/migration.clj

```diff
@@ -40,5 +57,6 @@
          :db/cardinality :db.cardinality/one
          :db/doc "Migration File Name"
          :db.install/_attribute :db.part/db}]
-       (d/transact connection)
-       deref))
+       (set-tx-instant tx-instant)
+       (->> (d/transact connection))
```

Comment:
This is a bit confusing, I'm on the fence if it's not better to use an anon `fn` or `#()` syntax here...